### ISSUE TITLE
[Mega Drive] Saving Issues for Korean FIFA Soccer 95 and NBA Live 95, Resolution of Audio Issues

### ISSUE THREAD

**Issue by HeroponRikiBestest** [higan-emu/ares#1052]
FIFA Soccer 95 (Korea) (En,Fr,De,Es) and NBA Live 95 (Korea) lack SRAM indicators in their headers, unlike their US/EU counterparts, breaking save functionality. However, editing their headers to include the SRAM functionality fixes the issue, and will cause both games to save properly. The no-intro dump of FIFA ( https://datomatic.no-intro.org/index.php?page=show_record&s=32&n=1622) includes a picture of the PCB (https://datomatic.no-intro.org/attachments%20-%20sources/Sega%20-%20Mega%20Drive%20-%20Genesis/20181124_191540_1622_pcb_front.jpg)  confirming that it does have SRAM included, so ares should include an entry in mia/medium/mega-drive.cpp in order to enable SRAM for it. Unfortunately, while the box for NBA Live 95 denotes it has save functionality (https://cdn.discordapp.com/attachments/976414249893117962/1088642874398232606/0vm6y75x.png, 플레이어의 개인성적을 기록하기 위해 배터리 백업 슬롯을 사용, "Use a battery backup slot to record a player's personal performance"), we don't actually have any picture of the PCB to confirm they did indeed include SRAM in the cartridge, so there's always the small chance that for whatever reason they did actually remove it from the cartridge, however unlikely. I don't know what the proper protocol is; given the evidence, I would say add an entry for it in mega-drive.cpp as well, but that's not up to me.

Whatever is chosen, the issues with the music for NBA Live 95 (Korea) noted in the compatibility list should be removed and the game set to Completable (Unverified), as the lack of audio after the title screen is due to the game defaulting all options to off with the lack of SRAM (including music, sound effects, etc), wheras the US/EU version defaults them to being on since SRAM is initialized properly. I will wait to change the compatibility database, in case either the developers want to do it themselves, and otherwise to denote that you need to go to the options menu ingame on every load to turn music/etc on if it's decided to not add an entry for the game.

```
file: FIFA Soccer 95 (Korea) (En,Fr,De,Es).md
sha256: <b6b1f9666adcccbefeb4bd67dbbbc666b96f4aee566bc459ce11b210f30950fe>

file: NBA Live 95 (Korea).md
sha256: <e67d99837dc8861d6f538d36d893d80ebdfb338a5dfb4f5a7c46735d0b4f68aa>
```

(For anyone wondering, I went through the rest of the Korean-exclusive library to check; the other 10 games either never had save functionality in any region, or have the SRAM properly set in the headers and save properly. It seems it was just the EA games that had this change made to their headers. All other mega drive games released in Korea share the same ROM as another region.)

**Comment by HeroponRikiBestest** [comment]
SRAM info starting at offset 432 on US/EU copies: `52 41 F8 20 00 20 00 01 00 20 3F`
Backup RAM Start: 00200001
Backup RAM End: 00203FFF

Above info same for both titles.

Info on chip shown on Korean FIFA:
![image](https://user-images.githubusercontent.com/50224630/227507689-32f0f0f7-8e6d-4732-82ad-7e25a10faf37.png)

Obviously impossible to confirm the chip on NBA Live for the moment.

**Comment by LukeUsher** [comment]
Fixed as of 4713fb37cc013c3adbbd9b0db959135da3aaab87
